### PR TITLE
DH-1399 Acceptance tests add users with other permissions

### PR DESCRIPTION
**Pull request by feedmypixel**
This work adds the following `users` and relevant `access tokens` to be used in [circleCi](https://circleci.com/workflow-run/cce8bff0-613f-4905-9b7c-78d5b9294794)

- `dit_staff@datahub.com`: `ditStaffToken`
- `lep_staff@datahub.com`: `lepStaffToken`
- `da_staff@datahub.com`: `daStaffToken`

### CODE REVIEW

File: setup-uat.sh [hunk 1]
```diff
@@ -1,32 +1,67 @@
...
 from oauth2_provider.models import AccessToken
 from datahub.company.models import Advisor
 
 dit_east_midlands_id = '9010dd28-9798-e211-a939-e4115bead28a'
 
-user = Advisor.objects.create_user(
-    email='${QA_USER_EMAIL}',
-    first_name='Circle',
-    last_name='Ci',
+ditStaffUser = Advisor.objects.create_user(
```
Comment (elcct) [inline setup-uat.sh]: Can we not use camelCase?

Comment (feedmypixel) [reply setup-uat.sh]: How would you like it?

Comment (reupen) [reply setup-uat.sh]: underscores_with_spaces for most things (so dit_staff_user for instance).

(The exception is class names/user-defined types, which use UpperCamelCase.)

File: setup-uat.sh [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
+)
+
+AccessToken.objects.create(
+    user=daStaffUser,
+    token='daStaffToken',
+    expires=now() + datetime.timedelta(days=1),
+    scope='data-hub:internal-front-end',
+)
+
+lep_coast_to_capital_id = '02f14b0b-5f25-e511-b6bc-e4115bead28a'
```
Comment (reupen) [inline setup-uat.sh]: I think the LEP and DA used should tie up with one of the ones in the fixtures. 

https://github.com/uktrade/data-hub-leeloo/blob/24280e0c59f67b40452775db7ebe56a1fc0a34c4/fixtures/test_data.yaml#L153-L199

So probably Heart of the South West LEP and Welsh Government (Investment) for now.

**Pull request merged**